ci/test-clang-docker.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ ! -d /repo ]]; then
+    echo "[-] This script needs to be run within Docker! Use test-clang.sh instead."
```

Comment:
Is that because Docker or because Fedora? Or the `:/repo:ro` in `docker run`?

Reply:
The latter. I assume everything is in `/repo`.

Would it be clearer to say "This script is expected to be run in Docker"?